doc/users/whats_new/js-animation.rst

```diff
@@ -0,0 +1,13 @@
+Merge JSAnimation
+-----------------
+
+Jake Vanderplas' JSAnimation package has been merged into matplotlib. This
+adds to matplotlib the `~matplotlib.animation.HTMLWriter` class for
+generating a javascript HTML animation, suitable for the IPython notebook.
+This can be activated by default by setting the ``animation.html`` rc
+parameter to ``jshtml``. One can also call the ``anim_to_jshtml`` function
+to manually convert an animation. This can be displayed using IPython's
+``HTML`` display class::
+
+    from IPython.display import HTML
+    HTML(animation.anim_to_jshtml(anim))
```

Comment:
From this, it isn't really clear if one could use HTMLWriter as a writer like any other registered writer class to save out to an html file, nor does it mention what the registered name is.